bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx122-na.html

```diff
@@ -0,0 +1,136 @@
+            <td><span class="compare-icon compare-icon-some">Somewhat</span></td>
+            <td><span class="compare-icon compare-icon-yes">Yes</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+          </tr>
+          <tr>
+            <th scope="row">Copy URL without site tracking</th>
+            <td><span class="compare-icon compare-icon-yes">Yes</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+          </tr>
+          <tr>
+            <th scope="row">Prevent sites from tracking and sharing data</th>
+            <td><span class="compare-icon compare-icon-yes">Yes</span></td>
+            <td><span class="compare-icon compare-icon-some">Somewhat</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+            <td><span class="compare-icon compare-icon-yes">Yes</span></td>
+            <td><span class="compare-icon compare-icon-some">Somewhat</span></td>
+            <td><span class="compare-icon compare-icon-some">Somewhat</span></td>
+          </tr>
+          <tr>
+            <th scope="row">Tracker content blocking</th>
+            <td><span class="compare-icon compare-icon-yes">Yes</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+            <td><span class="compare-icon compare-icon-yes">Yes</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+          </tr>
+          <tr>
+            <th scope="row">Enforce user’s choice to not let websites sell their data</th>
+            <td><span class="compare-icon compare-icon-yes">Yes</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+            <td><span class="compare-icon compare-icon-yes">Yes</span></td>
+            <td><span class="compare-icon compare-icon-no">No</span></td>
+          </tr>
+        </tbody>
+        <tfoot>
+          <tr>
+            <td colspan="7" role="presentation" class="compare-table-legend">
+              <span class="legend-label">Legend:</span>
+              <span class="compare-icon compare-icon-yes">Yes</span> Yes
+              <span class="compare-icon compare-icon-some">Yes</span> Somewhat
+              <span class="compare-icon compare-icon-no">Yes</span> No
+            </td>
```

Comment:
You could use a fieldset and legend tags here i think? Not 100% sure on it

Reply:
I think that's really just for forms, and alas this isn't that kind of legend. 🐉